guides/source/working_with_javascript_in_rails.md

```diff
@@ -141,6 +141,7 @@ follow this pattern.
 Built-in Helpers
 ----------------------
 
+If you are using Rails with the [Asset Pipeline](asset_pipeline.html) enabled, 
```

Comment:
This conditional feels too strong for me. The phrasing makes it very easy to infer that this is something you'd need to opt in to, whereas the truth is closer to "unless you've gone out of your way to turn it off, ...".

And that's where we run into an issue, because the above is true for almost every part of the framework: it's possible to turn it off, and if you turn it off, it'll stop working. The documentation will get pretty tedious if every section needs to carry such a disclaimer. :confused: